### PR TITLE
fix(textures): bumped reference lines to be on top of spell textures

### PR DESCRIPTION
**Pull request by sneakyverill**
Also add a top and bottom pixel adjust so reference lines are not on top of the visible border

### CODE REVIEW

File: EventHorizon/Frame/SpellFrameBuilder.lua [hunk 1]
```diff
@@ -56,13 +57,13 @@ function SpellFrameBuilder:WithSender()
...
 
 function SpellFrameBuilder:WithIcon()
-	self.icon = self.frame:CreateTexture(nil, "BORDER")
-	self.frame.icon = icon
-	self.frame.stacks = self.frame:CreateFontString(nil, "OVERLAY", "GameTooltipText")
-	self.frame.stacks:SetPoint("BOTTOMRIGHT",self.frame,"BOTTOMLEFT")
-	local texture = select(3,GetSpellInfo(self.spellId))
-	self.icon:SetTexture(texture)
-	self.icon:SetPoint("TOPRIGHT", self.frame, "TOPLEFT")
+	self.icon = self.frame:CreateTexture(nil, "BORDER")
```
Comment (DragonKarsh) [inline EventHorizon/Frame/SpellFrameBuilder.lua:60]: whats the change here? Looks like just added an empty line on ln 59?

File: EventHorizon/Frame/MainFrame.lua [hunk 2]
```diff
@@ -255,7 +255,7 @@ function MainFrame:UpdateAllFrames()
 	if self.nowReference then
 		self.nowReference
 		:GetTexture()
-		:SetPoint('TOPLEFT',self.frame,'TOPLEFT', -past/(future-past)*width, 0)	
+		:SetPoint('TOPLEFT',self.frame,'TOPLEFT', -past/(future-past)*width, -1)
```
Comment (DragonKarsh) [inline EventHorizon/Frame/MainFrame.lua:258]: why are we offsetting the y axis here?

File: EventHorizon/Frame/MainFrame.lua [hunk 3]
```diff
@@ -265,7 +265,7 @@ function MainFrame:UpdateAllFrames()
 	if self.gcdReference then
 		self.gcdReference
 		:GetTexture()
-		:SetPoint('TOP',self.frame,'TOP', -past/(future-past)*width-0.5+height, 0)	
+		:SetPoint('TOP',self.frame,'TOP', -past/(future-past)*width-0.5+height, -1)
```
Comment (DragonKarsh) [inline EventHorizon/Frame/MainFrame.lua:268]: Is there a reason we are moving these reference lines up 1 unit?

File: EventHorizon/Reference/Reference.lua [hunk 4]
```diff
@@ -12,8 +12,8 @@ setmetatable(Reference, {
 function Reference:new(frame)
 	self.frame = frame
 
-	self.texture = self.frame:CreateTexture(nil, 'BORDER')
-	self.texture:SetPoint('BOTTOM',self.frame,'BOTTOM')
+	self.texture = self.frame:CreateTexture(nil, 'OVERLAY')
+	self.texture:SetPoint('BOTTOM',self.frame,'BOTTOM', 0, 1)
```
Comment (DragonKarsh) [inline EventHorizon/Reference/Reference.lua:16]: why (0,1) and not (0,0)

File: EventHorizon/Frame/SpellFrameBuilder.lua (hunk 1 above)
Comment (sneakyverill) [reply EventHorizon/Frame/SpellFrameBuilder.lua:60]: yea new line probably. no change

File: EventHorizon/Frame/MainFrame.lua (hunk 2 above)
Comment (sneakyverill) [reply EventHorizon/Frame/MainFrame.lua:258]: as explained in DM

File: EventHorizon/Frame/MainFrame.lua (hunk 3 above)
Comment (sneakyverill) [reply EventHorizon/Frame/MainFrame.lua:268]: as explained in DM

File: EventHorizon/Reference/Reference.lua (hunk 4 above)
Comment (sneakyverill) [reply EventHorizon/Reference/Reference.lua:16]: as explained in DM and for all the rest above - border visibility cosmetic

**Pull request merged**